web/client/components/geostory/layouts/sections/GeoCarousel.jsx

```diff
@@ -72,6 +73,10 @@ const GeoCarousel = ({
     defaultMarkerStyle = {
         iconColor: 'cyan',
         iconShape: 'circle'
+    },
+    highlightedMarkerStyle = {
+        iconColor: 'red',
```

Comment:
```suggestion
        iconColor: 'green',
```
green is more understandable.
![image](https://user-images.githubusercontent.com/1279510/142463762-e7548051-70e4-4342-83ba-b9a7b143bcb6.png)